python/mujinplanningclient/packingplanningclient.py

```diff
@@ -56,20 +60,33 @@ def StartPackFormationComputationThread(self, timeout=10, debuglevel=4, toolname
         taskparameters.update(kwargs)
         return self.ExecuteCommand(taskparameters, toolname=toolname, timeout=timeout)
 
-    def StartSingleSkuPackFormationComputationThread(self, packingParameters, packLocationName, toolname=None, timeout=10, **kwargs):
+    def StartSingleSKUPackFormation(
+        self,
+        userPackFormationComputationParameters,
+        systemState,
+        toolName=None,
+        patternName=None,
+        timeout=None,
+        **kwargs
+    ):
+        # type: (dict[str, Any], dict[str, Any], Optional[str], Optional[str], Optional[float], Any) -> None
```

Comment:
```suggestion
    def StartSingleSKUPackFormation(
        self,
        userPackFormationComputationParameters,  # type: dict[str, Any]
        systemState,  # type: dict[str, Any]
        toolName=None,  # type: Optional[str]
        patternName=None,  # type: Optional[str]
        timeout=None,  # type: Optional[float]
        **kwargs  # type: Any
    ):  # type: (...) -> None
```
nit

Reply:
Interestingly, though `darker` split up the lines, it did not move the types up. I guess modern tooling doesn't understand ancient Python.

Reply:
Yes, and who could blame it. Pretty sure it would have worked in Python 3.